[2025-03-04T17:28:49.611+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-04T17:28:49.635+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ha_etl_process.load_task scheduled__2025-03-03T00:00:00+00:00 [queued]>
[2025-03-04T17:28:49.645+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ha_etl_process.load_task scheduled__2025-03-03T00:00:00+00:00 [queued]>
[2025-03-04T17:28:49.645+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-04T17:28:49.659+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load_task> on 2025-03-03 00:00:00+00:00
[2025-03-04T17:28:49.670+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ha_etl_process', 'load_task', 'scheduled__2025-03-03T00:00:00+00:00', '--job-id', '212', '--raw', '--subdir', 'DAGS_FOLDER/ha_etl_process.py', '--cfg-path', '/tmp/tmpmzas18v5']
[2025-03-04T17:28:49.671+0000] {standard_task_runner.py:105} INFO - Job 212: Subtask load_task
[2025-03-04T17:28:49.675+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=530) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-04T17:28:49.675+0000] {standard_task_runner.py:72} INFO - Started process 576 to run task
[2025-03-04T17:28:49.717+0000] {task_command.py:467} INFO - Running <TaskInstance: ha_etl_process.load_task scheduled__2025-03-03T00:00:00+00:00 [running]> on host 151d4861bf1b
[2025-03-04T17:28:49.807+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ha_etl_process' AIRFLOW_CTX_TASK_ID='load_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-03T00:00:00+00:00'
[2025-03-04T17:28:49.808+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-04T17:28:49.809+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-04T17:28:49.809+0000] {logging_mixin.py:190} INFO - Current task name:load_task state:running start_date:2025-03-04 17:28:49.635711+00:00
[2025-03-04T17:28:49.810+0000] {logging_mixin.py:190} INFO - Dag name:ha_etl_process and current dag run status:running
[2025-03-04T17:28:49.810+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-04T17:28:49.826+0000] {logging_mixin.py:190} WARNING - /opt/***/dags/ha_etl_process.py:186 FutureWarning: Passing literal json to 'read_json' is deprecated and will be removed in a future version. To read from a literal string, wrap it in a 'StringIO' object.
[2025-03-04T17:28:49.862+0000] {logging_mixin.py:190} INFO - âœ… Dados carregados com sucesso em /opt/***/dags/output/data.parquet
[2025-03-04T17:28:49.863+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-04T17:28:49.871+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-04T17:28:49.872+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=ha_etl_process, task_id=load_task, run_id=scheduled__2025-03-03T00:00:00+00:00, execution_date=20250303T000000, start_date=20250304T172849, end_date=20250304T172849
[2025-03-04T17:28:49.892+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-04T17:28:49.892+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-04T17:28:49.893+0000] {logging_mixin.py:190} INFO - Dag name:ha_etl_process queued_at:2025-03-04 17:28:41.027031+00:00
[2025-03-04T17:28:49.893+0000] {logging_mixin.py:190} INFO - Task hostname:151d4861bf1b operator:PythonOperator
[2025-03-04T17:28:49.930+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-04T17:28:49.958+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-04T17:28:49.961+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
